timemachine/fe/mle.py

```diff
@@ -279,7 +283,7 @@ def infer_node_vals_and_errs_networkx(
     edge_idxs = np.array(sg_relabeled.edges)
 
     dgs, dg_errs = infer_node_vals_and_errs(
-        edge_idxs,
+        np.array([e[:2] for e in edge_idxs]),
```

Comment:
I found `e[:2]` surprising. Is `len(e) != 2`?

Reply:
Will add a comment. With MultiDiGraph it's (from_node, to_node, edge_key) where edge_key = 0, 1, ... for each duplicate edge.

Reply:
Ahh, then might rename this from `edge_idxs` to `edges` (since depending on input type it may or may not just contain idxs)

Reply:
Updated in [65e7c57](https://github.com/proteneer/timemachine/pull/1064/commits/65e7c5797f2794ddae05eb632ed29678221ffc49)